frontend/src/components/user-service/account-settings-page/AdjustPrivilegesList.tsx

```diff
@@ -0,0 +1,117 @@
+'use client'
+
```

Comment:
What is this for @@?

Reply:
can remove, it doesn't really matter too much